src/components/RSVPControl.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'react-router'
+const { object, func } = React.PropTypes
+import { filter } from 'lodash'
+import Avatar from './Avatar'
+import Dropdown from './Dropdown'
+import { changeEventResponse } from '../actions'
+
+export default class RSVPControl extends React.Component {
+  static propTypes = {
+    currentUser: object,
+    post: object,
+    dispatch: func
+  }
+
+  render () {
+    var props = this.props
+    let { post, currentUser } = props
```

Comment:
I also need the props var for later in eventResponseClicked (can't use this.props because this is not bound to the same thing anymore)
Here I could do this instead: 
```
var props = this.props
let { post, currentUser } = this.props
```
your call